torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -120,10 +120,22 @@ PjRtComputationClient::PjRtComputationClient() {
     TF_VLOG(1) << "Initializing PjRt GPU client...";
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
     int local_rank = sys_util::GetEnvInt(env::kEnvPjRtLocalRank, 0);
+    
+    // if (dist_runtime_service_ != nullptr && local_rank == 0) {
+    //   std::string dist_service = "127.0.0.1:8547";
+    //   xla::CoordinationServiceImpl::Options options;
+    //   auto local_process_count = sys_util::GetEnvInt(env::kEnvPjRtLocalProcessCount, 1);
+    //   options.num_nodes = local_process_count;
+    //   dist_runtime_service_ = xla::GetDistributedRuntimeService(dist_service, options).value();
+    // }
+    
+    // std::string dist_service_addr =
+    //     sys_util::GetEnvString(env::kEnvPjrtDistServiceAddr, "");
     std::string dist_service_addr =
-        sys_util::GetEnvString(env::kEnvPjrtDistServiceAddr, "");
+        sys_util::GetEnvString("MASTER_ADDR", "")+":8547";
```

Comment:
The port number `8547` seems to be a hardcoded value, which could potentially lead to conflicts with other processes in a cloud environment. Is it possible to allow users to specify a different port number? Otherwise, they would need to recompile the torch_xla program.

Reply:
ditto